infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesDeployments.java

```diff
@@ -346,6 +353,8 @@ public void onClose(KubernetesClientException cause) {
    *     otherwise, it must be explicitly closed
    */
   public CompletableFuture<Void> waitRunningAsync(String name) {
+    final Span tracingSpan = tracerUtil.buildSpan("WaitRunningAsync", null, workspaceId, name);
```

Comment:
`Bootstrap` and `checkServers` spans are managed by `KubernetesInternalRuntime`. Why can not `KubernetesInternalRuntime` manage `WaitRunningAsync` span as well? Like
```java
    final Span tracingSpan = tracerUtil
        .buildSpan("WaitRunningAsync", null, getContext().getIdentity().getWorkspaceId(),
            machine.getName());
    CompletableFuture<Void> waitFuture =
        namespace.deployments().waitRunningAsync(machine.getPodName())
            .whenComplete((v, ex) -> {
              if (ex != null) {
                tracerUtil.finishSpanAsFailure(tracingSpan, ex.getMessage());
              } else {
                tracerUtil.finishSpan(tracingSpan);
              }
            });
```

Reply:
That's a great point, and I don't know why I didn't think of it. Thanks.